cosmos.v1.proto

```diff
@@ -0,0 +1,482 @@
+syntax = "proto3";
+
+package cosmos.v1;
+
+option go_package = "github.com/figment-networks/tendermint-protobuf-def/cosmos";
```

Comment:
do we build this entire file manually? Or is it a copy paste from the cosmos-sdk?

Reply:
it is copy paste from cosmos-sdk